site/dat/repo/blazemeter.json

```diff
@@ -733,4 +733,25 @@
       }
     }
   }
+  {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected character ('{' (code 123)): was expecting comma to separate Array entries](https://app.codacy.com/app/apc4/jmeter-plugins/pullRequest?prid=4661576)